src/RpcServer/RpcServer.csproj

```diff
@@ -11,9 +11,5 @@
       <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
     </None>
   </ItemGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.ResponseCompression" Version="2.2.0" />
```

Comment:
Did you test the compression?

Reply:
> Did you test the compression?

Not deeply, but tested RPC APIs(getblockcount, getblock, etc) work well.

Reply:
Could you check if the response it's compressed?

Reply:
> Could you check if the response it's compressed?

![image](https://user-images.githubusercontent.com/38493437/103076609-14a5eb00-4609-11eb-894a-543a117d4767.png)
Looks like compressed.

Reply:
Good!